terraform/modules/ecs_module/alb.tf

```diff
@@ -0,0 +1,46 @@
+resource "aws_alb" "main" {
```

Comment:
🚫 **[tfsec]** <[AVD-AWS-0052](https://aquasecurity.github.io/tfsec/v1.27.6/checks/aws/elb/drop-invalid-headers/)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Application load balancer is not set to drop invalid headers.